lib/compress/zstd_compress_internal.h

```diff
@@ -152,6 +166,7 @@ struct ZSTD_matchState_t {
     optState_t opt;         /* optimal parser state */
     const ZSTD_matchState_t* dictMatchState;
     ZSTD_compressionParameters cParams;
+    rawSeqStore_t* ldmSeqStore; /* reference to ldm seq store if there is one */
```

Comment:
Is it guaranteed to be properly initialized ?
(I presume that `ldmSeqStore == NULL` is a valid entry and means "ldm seqStore not present")